app/presenters/licences/base-licences.presenter.js

```diff
@@ -33,6 +33,20 @@ function formatAbstractionAmounts(licenceVersionPurpose) {
   return details
 }
 
+/**
+ * Pluralise a word based on a count
+ *
+ * When the count is 1, the word is returned unchanged. Otherwise, the word is appended with an 's'.
+ *
+ * @param {string }word
+ * @param {number} count
+ *
+ * @returns {string} The pluralised word
```

Comment:
The pluralised version of the word if `count` is greater than 1, else `word`.

Reply:
I was going to put this but that is not what the logic is doing. 

I can update the logic and the wording ?